main.js

```diff
@@ -0,0 +1,99 @@
+console.log('hi');
+
+//smokescreen says surprise in console after 2 seconds (2000 ms)
+var intervalId = window.setInterval(function(){
```

Comment:
This uses `setInterval` so it says "hi" __every__ 2 seconds. If you want something to happen __after__ two seconds use `setTimeout`